src/product.css

```diff
@@ -0,0 +1,84 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__photo {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.product-card__title-text {
+  font-size: 12px;
+  font-weight: 500;
+  margin-bottom: 4px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.product-card__code-product {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  margin-bottom: 16px;
+  color: #616070;
+}
+
+.product-card__raiting {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.product-card__raiting-comments {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.product-card__price-card {
+  color: #616070;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+}
+
+.product-card__price-card_price {
+  color: #060b35;
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+}
+
+.product-card__btn {
+  height: 40px;
+  width: 100%;
+  line-height: 40px;
+  font-weight: 700;
+  font-size: 14px;
+  text-align: center;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-style: 14px;
+  color: #fff;
+  background-color: #00acdc;
+  border-radius: 5px;
+}
+
+.product-card__btn:hover {
+  background-color: #fff;
+  color: #00acdc;
+  border: 1px solid #00acdc;
```

Comment:
fix 8th rule in the [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)